docs/src/pages/index.tsx

```diff
@@ -28,6 +29,8 @@ import SectionHeader from '../components/SectionHeader';
 import BlurredSection from '../components/BlurredSection';
 import '../styles/main.less';
 
+// @ts-ignore
```

Comment:
ehhhh, how bad is it? Should we fix, or sweep it under the rug?

Reply:
I was getting type errors on GitHubButton not being a JSX node and was like "shut up!" Personally for the docs Im not too worried on tsc complain, either it renders or it don't...

Reply:
For now will sweep as i really don't feel like spending another 15 minutes looking into this and clearly that's a false negative.